convert2rhel/subscription.py

```diff
@@ -22,7 +22,12 @@
 from collections import namedtuple
 from time import sleep
 
-from convert2rhel import backup, pkghandler, utils
+import dbus
+import dbus.connection
```

Comment:
```suggestion
```
Unless it's about some kind of optimization, it isn't necessary.

Reply:
I like to have this because whether this works: `import dbus ; dbus.connection` is an implementation detail of the dbus module.  It will work if `dbus/__init__.py` has something like:
``` python
from dbus import connection
```
If `dbus/__init__.py` does not have something like that line, then `import dbus.connection` would be necessary in our code.